components/dropdown/dropdown-tabs.js

```diff
@@ -25,17 +29,21 @@ class DropdownTabs extends DropdownContentMixin(LitElement) {
 		`;
 	}
 
-	_onMenuResize() {
```

Comment:
Similar to `d2l-dropdown-menu`, this logic needs to use the provided dimensions.  In addition, it must account for the height of the tabs, and whitespace.  This is needed so that the dropdown can animate/position/size itself.  Otherwise, it would need to wait until the menu animation is complete, or disable the animation, especially for nested menus.

Background - hierarchical-view (and in turn menu) dispatches this event, and it provides the measured dimensions of the content as a hint, which dropdown is intended to use, so that it can position/animate/size itself immediately.